feed2discord.py

```diff
@@ -892,51 +522,432 @@ async def background_check_feed(feed, asyncioloop):
         # No matter what goes wrong, wait same time and try again
         finally:
             logger.info(
-                feed +
+                self.feed +
                 ":sleeping for " +
-                str(rss_refresh_time) +
+                str(self.rss_refresh_time) +
                 " seconds")
-            await asyncio.sleep(rss_refresh_time)
+            await asyncio.sleep(self.rss_refresh_time)
+
+    async def skew_sleep_before_refresh(self):
```

Comment:
Maybe rename this to something like "sleep_before_first_run"? It's there so that at startup time, the feeds don't all hit at once, since that causes load issues and some feed sources hate if you hit them too fast.